model/src/main/kotlin/OrtResult.kt

```diff
@@ -153,6 +153,29 @@ data class OrtResult(
         resolvedConfiguration.packageConfigurations.orEmpty().groupBy { it.id }
     }
 
+    private val excludedAffectedPathIssuesForId: Map<Identifier, Set<Issue>> by lazy {
```

Comment:
Commit message: I'm having a hard time understanding what and why you are doing it.
Can you please rephrase?